ansible/roles/configure/tasks/acltb_permanent_config.yml

```diff
@@ -0,0 +1,62 @@
+#-----------------------------------------
+# Apply ACL configuration
+#-----------------------------------------
+- set_fact:
+    acltb_configs:
+      - "{{ 'acltb_persistent_acl_table' }}"
+      - "{{ 'acltb_persistent_acl_rules' }}"
+
+# Gather minigraph facts
+- name: Gathering minigraph facts about the device
+  minigraph_facts: host={{ inventory_hostname }}
+  become: no
+  connection: local
+
+- name: Print neighbors in minigraph
+  debug: msg="{{ minigraph_neighbors }}"
+
+- name: Read port reverse alias mapping
+  set_fact:
+    alias_reverse_map: "{{ lookup('file', 'roles/sonicv2/files/ssw/{{ sonic_hwsku }}/alias_reverse_map.json') | from_json }}"
+
+# Generate file with ACL table for rules which permit other tests traffic
+- template: src={{ item }}.j2 dest=/tmp/{{ item }}.json
+  connection: local
+  with_items:
+    - "{{ acltb_configs }}"
+
+# Copy ACL config to the swss container via the switch
+- name: Copy ACL config file to the DUT
+  copy: src="/tmp/{{ item }}.json" dest="/tmp/{{ item }}.json"
+  with_items:
+    - "{{ acltb_configs }}"
+
+- name: Copy ACL config files to the swss container
+  command: docker cp "/tmp/{{ item }}.json" swss:/etc/swss/config.d/"{{ item }}.json"
```

Comment:
>command: docker cp [](start = 2, length = 18)

If you follow the existing code 'ansible_shell_type', you don't need a 'docker cp' command. It is more ansible-friendly.